usaspending_api/database_scripts/matview_generator/mv_pre2008_award_search.json

```diff
@@ -28,7 +28,6 @@
     "  awards.total_subsidy_cost,",
     "  awards.total_loan_value,",
     "",
-    "  awards.recipient_id,",
```

Comment:
If you're updating the matviews, the models need to be updated as well
usaspending_api/awards/models_matviews/base_award_search.py LN 35